bin/stacks/routing-database-stack.ts

```diff
@@ -158,5 +163,20 @@ export class RoutingDatabaseStack extends cdk.NestedStack {
       billingMode: BillingMode.PAY_PER_REQUEST,
       timeToLiveAttribute: DynamoDBTableProps.TTLAttributeName,
     })
+
+    // Creates a Table for storing health state of RPC providers
+    this.rpcProviderHealthStateDynamoDb = new aws_dynamodb.Table(
+      this,
+      DynamoDBTableProps.RpcProviderHealthStateDbTable.Name,
+      {
+        tableName: DynamoDBTableProps.RpcProviderHealthStateDbTable.Name,
+        partitionKey: {
```

Comment:
no sort key ?

Reply:
i guess we can add timestamp to see latest changes right? or do we want to keep overriding?

Reply:
I don't see a downside for overriding. If we want to know what happened as history, we already published metrics for tracking all DB changes.

Reply:
we publish changes from tracking the metrics but not the DB changes right?

Reply:
After a DB change, we will publish a metrics about that change so we can remember it in future (the metrics is specifically for logging DB change).

Reply:
it depends if we ever considered queries that are chain wide or provider wide:
if you want to do `give me the status of all chain for a given provider`, then HK: provider, SK: chain
If you want to do `give the status of all providers for a given chain`, then HK: chain, SK: provider

Reply:
Here all keys are a `Chain-Provider` pair so the keys are flat. The number of such pairs will be well below 50 so there's really no concern for the query efficiency.

In reality, one provider may have different infra for serving different chains, the outage of one chain from this provider doesn't necessarily mean its other chain will be impacted. So I think tracking health state at `Chain-Provider` pair level is an reasonable choice.